Foundation/Dictionary.swift

```diff
@@ -18,7 +18,8 @@ extension Dictionary : _ObjectTypeBridgeable {
         
         var idx = 0
         
-        self.forEach { (keyItem, valueItem) in
+        self.forEach { (arg) in
```

Comment:
Sorry if this is not the best place to ask this question but what about overloading functions like `forEach` or `filter` to do something like https://gist.github.com/guidomb/b313667616612fc3e0c74c02a066dd8b and avoid having to unpack tuples